compressor/management/commands/compress.py

```diff
@@ -47,6 +49,10 @@ def add_arguments(self, parser):
                                  "(which defaults to STATIC_ROOT). Be aware that using this "
                                  "can lead to infinite recursion if a link points to a parent "
                                  "directory of itself.", dest='follow_links')
+        parser.add_argument('--processes', '-p', default=1, type='int',
+                            dest='processes',
+                            help='Number of parallel processes to use to compress blocks '
+                            '(default 1).'),
```

Comment:
I don't have a strong opinion one way or the other. I don't think backwards-compatibility matters here, if that's what you were worried about.